debian/changelog

```diff
@@ -1,3 +1,7 @@
+mercury (2.4.0~rc5-2) unstable; urgency=medium
+  [ Joseph Moore ]
+  * Add patch to na_ucx.c to check ep in key_resolve
+
```

Comment:
you're missing the date in the Debian log

Reply:
Fix it.